business_requirement/wizards/mail_compose_message.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models
+
+
+class MailComposeMessage(models.TransientModel):
+    _inherit = 'mail.compose.message'
+
+    def default_get(self, cr, uid, fields, context=None):
+        """ Handle composition mode. Some details about context keys:
+            - comment: default mode, model and ID of a record the user comments
+                - default_model or active_model
+                - default_res_id or active_id
+            - reply: active_id of a message the user replies to
+                - default_parent_id or message_id or active_id: ID of the
+                    mail.message we reply to
+                - message.res_model or default_model
+                - message.res_id or default_res_id
+            - mass_mail: model and IDs of records the user mass-mails
+                - active_ids: record IDs
+                - default_model or active_model
+        """
+        if context is None:
+            context = {}
+        result = super(MailComposeMessage, self).default_get(
+            cr, uid, fields, context=context)
+
+        # v6.1 compatibility mode
```

Comment:
This shouldn't go here in this module. If you have found a problem in the composition wizard, put a PR/issue against Odoo/OCB or make a specific patch module that deals with it, but don't insert this in a module about "Business requirements"